clients/kinesis.d.ts

```diff
@@ -992,7 +992,7 @@ declare namespace Kinesis {
      */
     MillisBehindLatest: MillisBehindLatest;
   }
-  export type SubscribeToShardEventStream = EventStream<{SubscribeToShardEvent?:SubscribeToShardEvent,ResourceNotFoundException?:ResourceNotFoundException,ResourceInUseException?:ResourceInUseException,KMSDisabledException?:KMSDisabledException,KMSInvalidStateException?:KMSInvalidStateException,KMSAccessDeniedException?:KMSAccessDeniedException,KMSNotFoundException?:KMSNotFoundException,KMSOptInRequired?:KMSOptInRequired,KMSThrottlingException?:KMSThrottlingException,InternalFailureException?:InternalFailureException}>;
+  export type SubscribeToShardEventStream = EventStream<{SubscribeToShardEvent?:SubscribeToShardEvent}>;
```

Comment:
Just to be sure, if you run `node ./scripts/typings-generator.js` is this what the file looks like?

Reply:
yes.